operations/kubernetes/Cert_Renewal_for_Kubernetes_and_Bare_Metal_EtcD.md

```diff
@@ -188,9 +220,9 @@ Check the expiration of the certificates.
     -rw------- 1 root root 1675 Sep 22 17:13 server.key
     ```
 
-   As we can see not all the certificate files were updated.
+   Not all the certificate files were updated in this example.
 
-   `IMPORTANT:` Some certificates were not updated because they have a distant expiration time and did not need to be updated. ***This is expected.***
+   **IMPORTANT:** Some certificates were not updated because they have a distant expiration time and did not need to be updated. ***This is expected.***
```

Comment:
```suggestion
   **IMPORTANT:** Some certificates were not updated because they have a distant expiration time and did not need to be updated. **This is expected.**
```